bigflow/bigquery/dataset_manager.py

```diff
@@ -285,6 +287,9 @@ def write(self, table_id: str, sql: str, mode: str) -> 'google.cloud.bigquery.ta
         job_config.destination = table_id
         job_config.write_disposition = mode
 
+        if self.job_labels:
+            job_config.labels = self.job_labels
```

Comment:
You could skip such if statements in many places if job_labels would be initialized with empty dict, not None